Sources/MuxPlayerSwift/PublicAPI/Options/PlaybackOptions.swift

```diff
@@ -24,7 +24,42 @@ public enum ResolutionTier {
     case upTo2160p
 }
 
-extension ResolutionTier {
+/// The min resolution tier you'd like your asset to be streamed at
+public enum MinResolutionTier {
+    /// By default no resolution tier is specified and Mux
+    /// selects the optimal resolution and bitrate based on
+    /// network and player conditions.
+    case `default`
+    /// The asset will stream with a resolution that does
+    /// not exceed 480p (640 x 480)
+    case upTo480p
```

Comment:
I'm wondering if we should use `atLeast` for minimum resolution instead of `upTo`. Any opinions? cc @muxinc/player-and-sdks